english/eating-walls/light-fire/fire.md

```diff
@@ -0,0 +1,4 @@
+You build a camp-fire, grab a fistful of the wall and put it on a stick.
```

Comment:
I can't seem to find where to go to start the story. Where do I go to start
language.md?
Thank you.
William F. Baur

On Fri, Oct 30, 2015 at 7:35 PM, Casey Collaborator <
notifications@github.com> wrote:

> In english/eating-walls/light-fire/fire.md
> <https://github.com/udacity/create-your-own-adventure/pull/2674#discussion_r43562811>
> :
>
> > @@ -0,0 +1,4 @@
> > +You build a camp-fire, grab a fistful of the wall and put it on a stick.
>
> It looks like this page is not reachable from the rest of the story. That
> is, if a reader starts in language.md and keeps clicking links, they will
> never find this page. You can fix this by finding a good place in the
> existing story to add an option with a link that leads to this page.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/udacity/create-your-own-adventure/pull/2674/files#r43562811>
> .
>



-- 
Sincerely,
William F. Baur